Packages/Sources/RootView/Home.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  SwiftUIView.swift
+//
+//
+//  Created by HH on 09/01/2024.
+//
+
+import NodeCryptoCore
+import SwiftUI
+
+struct Home: View {
+  @State private var activeTab: Tab = .search
+  @Namespace var animation
+
+  var body: some View {
+    VStack(spacing: 0) {
+      TabView(selection: $activeTab) {
+        Color.red
+          .tag(Tab.home)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Services")
+          .tag(Tab.search)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Notifications")
+          .tag(Tab.notifications)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Profile")
+          .tag(Tab.profile)
+          .toolbar(.hidden, for: .tabBar)
+      }
+
+      CustomTabBar()
+    }
+  }
+
+  @ViewBuilder
+  func CustomTabBar(tint: Color = .blue, inactiveTint: Color = .blue) -> some View {
```

Comment:
Wondering, why haven't you used the builtin tabBar/tabItem, like e.g. here https://www.hackingwithswift.com/quick-start/swiftui/adding-tabview-and-tabitem

Reply:
I was just trying to match the design and animate the circle to the selected tab. 
<img width="461" alt="Screenshot 2024-01-17 at 12 17 07 AM" src="https://github.com/hilalalhakani/NodeCrypto/assets/24891448/c48fc553-92d6-4aa2-b2f5-41758ecbf54e">